scripts/dependencies/tools.ts

```diff
@@ -386,34 +396,17 @@ export class RancherDashboard implements Dependency, GitHubDependency {
   }
 
   async getAvailableVersions(): Promise<string[]> {
-    const versions = await getPublishedReleaseTagNames(this.githubOwner, this.githubRepo);
+    const tagNames = await getPublishedReleaseTagNames(this.githubOwner, this.githubRepo);
 
-    // Versions that contain .plugins. exist solely for testing during
-    // plugins development. For more info please see
-    // https://github.com/rancher-sandbox/rancher-desktop/issues/3757
-    return versions.filter(version => !version.includes('.plugins.'));
+    return tagNames.map((tagName: string) => tagName.replace(/^desktop-v/, ''));
```

Comment:
Do we still need to filter out `.plugins.`? Why not?

Reply:
IIUC the old idea was to turn the dashboard into a plugin and have Rancher Desktop be a shell that this would plug into.

It is my understanding that that approach has been abandoned, and the code in the new repo is a "standalone extension". So the new repo will only have releases of this extension. @rak-phillip can correct me if I'm wrong.

Reply:
> It is my understanding that that approach has been abandoned, and the code in the new repo is a "standalone extension". So the new repo will only have releases of this extension. @rak-phillip can correct me if I'm wrong.

You are correct.